src/static/images/intro-background-fit.svg

```diff
@@ -0,0 +1 @@
+<svg id="BG" xmlns="http://www.w3.org/2000/svg" width="1920" height="1250"><defs><style>.cls-1{fill:#a8caba;fill-rule:evenodd;opacity:.05}</style></defs><path id="Rectangle_5_copy_4" data-name="Rectangle 5 copy 4" class="cls-1" d="M792 202h2088v651L792 1213V202z" transform="translate(-1023 -113)"/><path id="Rectangle_5_copy_5" data-name="Rectangle 5 copy 5" class="cls-1" d="M553 352h2088v651L553 1363V352z" transform="translate(-1023 -113)"/><path id="Rectangle_5_copy" data-name="Rectangle 5 copy" d="M879 94h2088v651L879 1105V94z" transform="translate(-1023 -113)" fill="#1a2b49" opacity=".7" fill-rule="evenodd"/><path class="cls-1" d="M553 30h2088v651L553 1041V30z" transform="translate(-1023 -113)"/><path id="Rectangle_5_copy_2" data-name="Rectangle 5 copy 2" d="M791 65h2088v651L791 1076V65z" transform="translate(-1023 -113)" fill="#f2f2f2" opacity=".05" fill-rule="evenodd"/><path id="Rectangle_5_copy_3" data-name="Rectangle 5 copy 3" d="M-403-258h2088v651L-403 753V-258z" transform="translate(-1023 -113)" opacity=".1" fill="#f2f2f2" fill-rule="evenodd"/></svg>
```

Comment:
Nit: Could this be pretty printed? It'd be easier to spot ways to optimize it.

Reply:
This is already optimized using SVGO https://jakearchibald.github.io/svgomg/ but sure, we could pretty-print it if necessary. We're using its base64 version anyway, but I've added in the images folder as a reference.

Reply:
Yeah I'm just curious if we can remove some ineffective attributes like `data-name`, `id`, or `full-rule`.

Reply:
You were right, I was able to further optimize it. Thanks for the heads up!